app/models/charging.rb

```diff
@@ -1,4 +1,4 @@
 class Charging < ApplicationRecord
   belongs_to :car
-  monetize :amount_cents, numericality: { greater_than: 0 }
+  monetize :amount_cents, numericality: { in: 0.01..4294967295 }
```

Comment:
Since it's in cents it should be in 1.. 4294967295, because 0.01 cent is actually  0.0001 USD...

Reply:
It's quite right, but in practice it's acting a bit different. When range starts form 1 in the form you are allowed to enter only integers and the model don't accept 0.5 for example.